modules/apps/collaboration/microblogs/microblogs-web/src/main/java/com/liferay/microblogs/web/internal/social/MicroblogsActivityInterpreter.java

```diff
@@ -102,7 +103,9 @@ else if (microblogsEntry.getType() ==
 			}
 		}
 
-		sb.append(HtmlUtil.escape(microblogsEntry.getContent()));
+		sb.append(
```

Comment:
Hey @brianikim why is this line different to https://github.com/liferay/liferay-portal/blob/master/modules/apps/collaboration/microblogs/microblogs-web/src/main/resources/META-INF/resources/microblogs/view_microblogs_entries.jsp#L86 ?
In this line we are escaping and in the jsp we are not. That seems like a bug. In the jsp we are using replaceNewLine and here we are not. 
Can you please check?

Reply:
Hello @sergiogonzalez ,

I checked the issue and confirmed that HtmlUtil.escape is not necessary in this case. I originally kept the code for the sake of consistency, but upon checking how both activity and microblogs portlet work, I have found that it is not necessary to include HtmlUtil.escape as HTML elements do not take effect in the portlets. I have resent you a PR here: https://github.com/sergiogonzalez/liferay-portal/pull/4272